openhands/events/stream.py

```diff
@@ -16,6 +16,7 @@
 class EventStreamSubscriber(str, Enum):
     AGENT_CONTROLLER = 'agent_controller'
     SECURITY_ANALYZER = 'security_analyzer'
+    RESOLVER = 'openhands_resolver'
     SERVER = 'server'
     RUNTIME = 'runtime'
     MAIN = 'main'
```

Comment:
If we want to support use of OpenHands as a library and allow downstream code to subscribe to the event stream, then maybe this should not be an enum but rather just be a string. This could either be fixed in this PR or we could open an issue and fix it in a future PR (possibly with OpenHands).

Reply:
We could also just get rid of this entirely. I put it here for safety (so you don't subscribe multiple times) but maybe it's not that useful